hole-punching-tests/Dockerfile

```diff
@@ -1,18 +1,13 @@
 # syntax=docker/dockerfile:1.5-labs
-FROM rust:1.72.0 as builder
 
-RUN rustup target add x86_64-unknown-linux-musl
-RUN --mount=type=cache,target=/var/cache/apt apt-get update && apt-get install -y musl-dev musl-tools
+FROM --platform=$BUILDPLATFORM rust:1.72.0 as builder
 
-# Run with access to the target cache to speed up builds
-WORKDIR /workspace
-ADD . .
-RUN --mount=type=cache,target=./target \
-    --mount=type=cache,target=/usr/local/cargo/registry \
-    cargo build --release --package hole-punching-tests --target x86_64-unknown-linux-musl
+ARG BUILDPLATFORM
 
-RUN --mount=type=cache,target=./target \
-    mv ./target/x86_64-unknown-linux-musl/release/hole-punching-tests /usr/local/bin/hole-punching-tests
+WORKDIR /workspace
+COPY . /workspace
+COPY /build.sh /build.sh
```

Comment:
I don't think this is necessary because `build.sh` is included by copying the entire repository, right?